player/lua/stats.lua

```diff
@@ -82,6 +84,15 @@ local o = {
 }
 options.read_options(o)
 
+o.term_width_limit = tonumber(o.term_width_limit) or -1
+o.term_height_limit = tonumber(o.term_height_limit) or -1
+if o.term_width_limit < 0 then
+    o.term_width_limit = nil
+end
```

Comment:
nit, could also be a one liner

Reply:
But that's not in line with the code style of the rest of the file.
Although there are two one liner `if`s, so there is precedent...